app/src/screens/Splash.tsx

```diff
@@ -386,17 +386,19 @@ const Splash = () => {
           await poolService.refreshPoolConnections()
           // eslint-disable-next-line @typescript-eslint/ban-ts-comment
           // @ts-ignore:next-line
```

Comment:
Can these comments be removed now that the patch has been removed?

Reply:
Not changed as part of the PR so I'd ignore 'em for now.